WiseBuy/View Controllers/ComposeViewController.m

```diff
@@ -20,9 +19,18 @@ @interface ComposeViewController () <UITextViewDelegate>
 @implementation ComposeViewController
 
 - (void)viewDidLoad {
-    [super viewDidLoad];
-    // Do any additional setup after loading the view.
+    UITapGestureRecognizer *tapGestureRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self
+                                                                          action:@selector(dismissKeyboard)];
+    [self.view addGestureRecognizer:tapGestureRecognizer];
 }
+
+- (void)dismissKeyboard {
+    [self.itemName resignFirstResponder];
```

Comment:
Would `[self.view endEditing:YES];` work? I saw this from https://stackoverflow.com/a/17467892

I would avoid resigning each text field individually, if possible, for the sake of code maintainability. It's very easy to make a coding mistake. If a future developer adds a new text field and forget to add it to this method, then dismissing the keyboard won't work from that field.

Reply:
ah thank you!